safe_grid_agents/parsing/parse.py

```diff
@@ -55,25 +53,32 @@
 }
 
 # YAML conversion helper
-type_map = {"float": float, "int": int, "str": str}  # Dict[str, type]
+TYPE_MAP = {"float": float, "int": int, "str": str}  # Dict[str, type]
 
 
-def map_type(x):
+def map_type(x: str) -> type:
     try:
-        return type_map[x]
+        return TYPE_MAP[x]
     except KeyError:
         return x
 
 
-def handle_parser_args(parsers, name, configs):
-    """Assist adding arguments from `configs` to parser `name` from collection `parsers`."""
-    p = parsers[name]
-    config = configs[name]
+def handle_parser_args(parsers, name, configs) -> None:
+    """Assist adding arguments from `configs` to parser `name` from collection
+    `parsers`."""
+    p, config = parsers[name], configs[name]
+
     try:
+        # `list` is necessary since we pop the original config keys and we
+        # can't use `deepcopy` as `dict.keys()` is a generator and is therefore
+        # unpickleable.
         for key in list(config.keys()):
             argattrs = {k: map_type(v) for k, v in config.pop(key).items()}
-            alias = argattrs.pop("alias")
-            p.add_argument("-{}".format(alias), "--{}".format(key), **argattrs)
+            if "alias" in argattrs:
+                alias = argattrs.pop("alias")
+                p.add_argument("-{}".format(alias), "--{}".format(key), **argattrs)
+            else:  # some arguments have no short form
```

Comment:
I think the goal was to have a short form for every argument, can you point to any specific args that don't have one?

Reply:
No args have only a long form (currently), but I ran into this when I only had a long form for `--tune`, and it was really unexpected behavior that required me to debug the crash.

It's a weird assumption to have hardcoded into the parser, and goes against most CLI's expected behavior.

Also, I think the parser should work in more cases, not less.

Reply:
It's also not written anywhere that having short forms for every argument is required, nor do I think it should be.